Tests/test_NCBITextParser.py

```diff
@@ -167,8 +170,7 @@ def test_text_2010L_blastp_001(self):
         self.assertAlmostEqual(record.blast_cutoff[1], 28.2)
 
     def test_text_2010L_blastp_002(self):
-        "Test parsing BLASTP 2.0.10 output without hits (text_2010L_blastp_002)"
-
+        """Test parsing BLASTP 2.0.10 output without hits (text_2010L_blastp_002)."""
```

Comment:
In a case like this, where the line length is only slightly above 80 characters, putting the one-line comment on an extra line would keep the line length under 80 characters.

```python
        """
        Test parsing BLASTP 2.0.10 outpu without hits (text_2010L_blastp_002).
        """
```

Do you think this is worth considering? (What would ```black``` do?)

Reply:
I'm happy to leave it - we don't current enforce a strict 80 limit (indeed the PEP8 wording is flexible on this), and also black uses 88 characters.

That said, I did rename some of the tests docstrings from "Test parsing..." to "Parsing..." and that would help here.

Reply:
Shorted by removing "Test " from the start (done for the whole file).